### PR TITLE
Add Support for Xilinx KCU116

### PR DESCRIPTION
**Pull request by mander1000**
Hey

I added support for the Xilinx KCU116 board (Kintex Ultrascale Plus).

I tested it to flash both SPI flash chips on the board (QSPI), which worked fine.

Feel free to comment and let me know if you need anything else.

mander

### CODE REVIEW

File: spiOverJtag/build.py [hunk 1]
```diff
@@ -127,15 +131,22 @@
...
                     'default': 1}
             elif part == "xcvu37p-fsvh2892":
                 tool_options = {'part': part + '-2L-e'}
+            elif part == "xcku5p-ffvb676":
+                tool_options = {'part': part + '-2-e'}
+                parameters["secondaryflash"]= {
+                    'datatype': 'int',
+                    'paramtype': 'vlogdefine',
+                    'description': 'secondary flash',
+                    'default': 1}
```
Comment (trabucayre) [inline spiOverJtag/build.py:149]: I'm not familiar about speedgrade for ultrascale+ but `-1` or `-2` are similar to equivalent for artix (for example)?
If so is it possible to use a low speed grade with a FPGA with highest grade?
The idea is to merge xcvu and xcku in the same test.
If I have true could you modify it (no need to rebuild the bitstream).
Thanks

Comment (mander1000) [reply spiOverJtag/build.py]: The naming for the speed grade is the same on all UltraScale devices (and I think even on all Xilinx devices). `-1` is the slowest. So sure, I changed that. I combined the family name to `Xilinx UltraScale` as it checks for `Virtex` and `Kintex` devices. Hope that is ok.

**Comment by mander1000** [comment]
should all be done now and ready to merge. :)

**Comment by trabucayre** [comment]
Applied. Thanks!

**Pull request merged**